src/System.Management.Automation/help/UpdateHelpCommand.cs

```diff
@@ -375,7 +375,42 @@ internal override bool ProcessModuleWithCulture(UpdatableHelpModuleInfo module,
                         // Gather destination paths
                         Collection<string> destPaths = new Collection<string>();
 
-                        destPaths.Add(module.ModuleBase);
```

Comment:
The logic at line 350 might have to be revised.
if (Utils.IsUnderProductFolder(module.ModuleBase) && (!Utils.IsAdministrator()))
{
    string message = StringUtil.Format(HelpErrors.UpdatableHelpRequiresElevation);
    ProcessException(module.ModuleName, null, new UpdatableHelpSystemException("UpdatableHelpSystemRequiresElevation",
            message, ErrorCategory.InvalidOperation, null, null));
    return false;
}

Reply:
Fixed.

Reply:
@adityapatwardhan : As per our conversation, we need to update UpdatableHelpRequiresElevation to include the UpdateHelpScope change.